src/routes/usersRoute.js

```diff
@@ -0,0 +1,53 @@
+const express = require('express');
+const {
+  httpSignupUser,
+  httpActivateUser,
+  httpLoginUser,
+  httpLogoutUser,
+  httpRefreshUserToken,
+  httpCheckAuthStatus,
+  httpUserValidation,
+  httpRestorePassword,
+  httpChangePassword,
+  passport,
+} = require('../controllers/users.controller');
+
+const usersRoute = express.Router();
+
+usersRoute.get(
+  '/login/google',
+  passport.authenticate('google', {
+    scope: ['email', 'profile'],
+    accessType: 'offline',
+    prompt: 'consent',
+  }),
+);
+
+usersRoute.get(
+  '/login/google/callback',
+  passport.authenticate('google', {
+    failureRedirect: '/login',
```

Comment:
You can move all routes in constants then use it in all places in your app

Reply:
how this should be looked like?)